lib/cron_parser.rb

```diff
@@ -295,4 +304,16 @@ def validate_source
       raise ArgumentError, 'not a valid cronline'
     end
   end
+
+  def strict_match?
+    @strict_match
+  end
+
+  def set_strict_match(strict_match = false)
+    @strict_match = false
+    if strict_match && [time_specs[:dom].last, time_specs[:dow].last].all?{|v| v != '*'}
```

Comment:
Is that check even needed? I mean if the user tells us to use strict matching, why would we refuse that? I'd rather do:

```ruby
  attr_accessor :strict_match
  alias_method :strict_match?, :strict_match
```

This would even get rid of the predicate method